src/client/src/ui/spotTile/components/SpotTile.tsx

```diff
@@ -1,134 +1,68 @@
-    const hasNotification = !!notification
-    const className = classnames('spot-tile', {
-      'spot-tile--stale':
-        (!pricingConnected || (price && price.priceStale)) &&
-        !(hasNotification && notification.notificationType === NotificationType.Trade),
-      'spot-tile--readonly': !executionConnected,
-      'spot-tile--executing': isTradeExecutionInFlight,
-      'spot-tile--error': hasNotification && notification.hasError
-    })
+interface State {
+  notional: string
+}
 
-    return className
+export default class SpotTile extends PureComponent<Props, State> {
+  state = {
+    notional: '1000000'
   }
 
-  createSpotTileControls() {
-    const { onPopoutClick, undockTile, displayCurrencyChart, isRunningOnDesktop, spotTileData, tornOff } = this.props
+  updateNotional = (notional: string) => this.setState({ notional })
 
-    return (
-      <SpotTileControls
-        onPopoutClick={onPopoutClick}
-        currencyChartIsOpening={spotTileData.currencyChartIsOpening}
-        displayCurrencyChart={displayCurrencyChart}
-        isRunningOnDesktop={isRunningOnDesktop}
-        undockTile={undockTile}
-        tornOff={tornOff}
-      />
-    )
-  }
+  executeTrade = (direction: Direction) => this.props.executeTrade(direction, numeral(this.state.notional).value())
 
-  createPriceComponents() {
-    const { currencyPair, spotTileData, executeTrade } = this.props
-    const title = `${currencyPair.base} / ${currencyPair.terms}`
-    if (spotTileData === null) {
-      return null
-    }
-
-    return (
-      <PriceControlsView
-        currencyPair={currencyPair}
-        title={title}
-        priceData={spotTileData.price}
-        executeTrade={executeTrade}
-      />
-    )
-  }
+  render() {
+    const { currencyPair, spotTileData, children } = this.props
+    const { notional } = this.state
 
-  getSpotTileContent() {
-    const { spotTileData, currencyPair } = this.props
-    const hasNotification = !!spotTileData.notification
-    const notionalInputClass = classnames('spot-tile__notional', {
-      hide: hasNotification
-    })
-    const spotDateClass = classnames('spot-tile__delivery', {
-      hide: hasNotification
-    })
-    const formattedDate = spotTileData.price ? spotDateFormatter(spotTileData.price.valueDate, false) : ''
+    const priceData = spotTileData && spotTileData.price
+    const spotDate = priceData && spotDateFormatter(priceData.valueDate, false).toUpperCase()
 
     return (
-      <div>
-        <span className="spot-tile__execution-label">Executing</span>
-        {this.createPriceComponents()}
-        <NotionalContainer className={notionalInputClass} currencyPair={currencyPair} />
-        <div className={spotDateClass}>
-          <span className="spot-tile__tenor">SP</span>
-          <span className="spot-tile__delivery-date">. {formattedDate}</span>
-        </div>
-      </div>
+      <SpotTileWrapper className="spot-tile-container">
```

Comment:
Because we're not using [babel-plugin-emotion], therefore we can use the component as a selector.
```js
const SpotTileWrapper = styled.div`
  ${SpotTileContainer} {
    ... magic
  }
`
```